### PR TITLE
Pass `--module-version` and `--root-url` parameters to `jazzy`

### PR DESCRIPTION
**Pull request by norio-nomura**
### Motivation:

It would be good if the API document was distributed as a docset that can be installed in Dash, like [Yams](https://www.jpsim.com/Yams/).

### Modifications:

Pass `--module-version` and `--root-url` parameters to `jazzy`.

### Result:

Jazzy generates docsets and add "Install in Dash" link to `index.html`.

### CODE REVIEW

**Comment by weissi** [comment]
@swift-nio-bot test this please

**Comment by tomerd** [comment]
@swift-nio-bot test this please

File: scripts/generate_docs.sh [hunk 1]
```diff
@@ -72,6 +72,8 @@ done
 
 for module in "${modules[@]}"; do
   args=("${jazzy_args[@]}"  --output "$root_path/docs/$version/$module" --module "$module")
+  args+=(--module-version $version
+         --root-url "https://apple.github.io/swift-nio/docs/$version/$module")
```
Comment (tomerd) [inline scripts/generate_docs.sh]: overall looks good, i think this can be merged with the line above?

Comment (norio-nomura) [reply scripts/generate_docs.sh]: done!

**Comment by weissi** [comment]
@swift-nio-bot test this please

**Comment by tomerd** [comment]
@swift-nio-bot test this please

**Comment by weissi** [comment]
@swift-nio-bot test this please

**Comment by norio-nomura** [comment]
Thanks! 🙏

**Pull request merged**